### PR TITLE
Add API Platform getting started

### PR DESCRIPTION
**Pull request by ChristopheDujarric**

### CODE REVIEW

File: docs/help/getting-started/index.md [hunk 1]
```diff
@@ -1,56 +1,11 @@
...
-
-We do not stop here. Feel free to browse this documentation or directly reach us to find out how to make the most of Bump.sh.
-There's a lot more to cover: from Hubs that groups several API documentation to Branching, hosting several version of a same doc, you'll find many simple options that make a huge change.
-
-The team is always here to listen if you have any feature requests or suggestions. We are here to help.
-
-**We hope you'll enjoy using Bump.sh!**
+- [Get started here](/quick-start) if you work directly with an OpenAPI or AsyncAPI file.
+- [Get started here](/api-platform) if you are building an API with API Platform (PHP).
+- [Get started here](/fastapi) if you are building an API with FastAPI (Python).
```
Comment (antlio) [inline docs/help/getting-started/index.md:11]: Would that make sense to expose the techno first and then state that we use x or y tool to use?
Like:
- PHP Quickstart
- Python Quickstart  (could by FastAPI Quickstart too, just wondering which one speak to a broader audience)
- etc.

Comment (ChristopheDujarric) [reply docs/help/getting-started/index.md:11]: Yes!

File: docs/help/getting-started/index.md [hunk 2]
```diff
@@ -1,56 +1,11 @@
...
-
-## And way more
-
-We do not stop here. Feel free to browse this documentation or directly reach us to find out how to make the most of Bump.sh.
-There's a lot more to cover: from Hubs that groups several API documentation to Branching, hosting several version of a same doc, you'll find many simple options that make a huge change.
-
-The team is always here to listen if you have any feature requests or suggestions. We are here to help.
-
-**We hope you'll enjoy using Bump.sh!**
+- [Quick start with specification files](getting-started/quick-start) if you work directly with an OpenAPI or AsyncAPI file.
```
Comment (paulRbr) [inline docs/help/getting-started/index.md:9]: we don't use the word `specification file` for a `definition file`. At least that's what we've talked about here: https://bump.sh/blog/what-is-openapi#definitions

Comment (ChristopheDujarric) [reply docs/help/getting-started/index.md:9]: Thanks!

File: docs/help/getting-started/api-platform.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Deploying docs from API Platform
+
+[API Platform](https://api-platform.com/) is an API-first PHP framework that allows you to create REST and GraphQL APIs using PHP classes, and automatically generates their documentation, based on the OpenAPI specification. If you are new to API Platform, check out [their getting started guide](https://api-platform.com/docs/distribution/).
```
Comment (jayk-u) [inline docs/help/getting-started/api-platform.md:3]: Shall we mention GraphQL?

File: docs/help/getting-started/index.md [hunk 4]
```diff
@@ -1,56 +1,11 @@
-# Getting Started
+# Getting started with Bump.sh
 
-Bump.sh connects people with APIs. From your API specifications, Bump.sh generates the documentation you need and offers many options to make it your custom and carefully crafted API experience.
+Bump.sh connects people with REST and event-driven APIs, by centralizing a reference point: their documentation, highlighted with the latest changes and updates. Customize their look and feel, manage who can access (read or maintain) which doc.
 
-## Create your first documentation
+From OpenAPI and AsyncAPI specifications, Bump.sh generates the documentation you need and offers many options to make it your custom and carefully crafted API experience. Those specifications are also supported by a wide variety of frameworks in various programming languages, which generate contract files automatically.
```
Comment (jayk-u) [inline docs/help/getting-started/index.md:5]: A bit too much marketing. This is a help center article, we do not need to push the product that much.

File: docs/help/getting-started/index.md [hunk 5]
```diff
@@ -1,56 +1,11 @@
...
-
-## And way more
-
-We do not stop here. Feel free to browse this documentation or directly reach us to find out how to make the most of Bump.sh.
-There's a lot more to cover: from Hubs that groups several API documentation to Branching, hosting several version of a same doc, you'll find many simple options that make a huge change.
-
-The team is always here to listen if you have any feature requests or suggestions. We are here to help.
-
-**We hope you'll enjoy using Bump.sh!**
+- [Quick start with definition files](quick-start) if you work directly with an OpenAPI or AsyncAPI file.
```
Comment (jayk-u) [inline docs/help/getting-started/index.md:9]: I know that's the solution we agreed on to... ease the changes for everyone.
However, I'm still sad to add an extra page and click for the users. I strongly believe that's a downgrade.

I won't come back on that topic so far, just want to put this here for the record.

**Review by jayk-u — APPROVED** [review]

File: docs/help/getting-started/api-platform.md (hunk 3 above)
Comment (ChristopheDujarric) [reply docs/help/getting-started/api-platform.md:3]: Good question. This is straight up coming from the API Platform description on their site.

File: docs/help/getting-started/index.md [hunk 6]
```diff
@@ -1,56 +1,11 @@
-# Getting Started
+# Getting started with Bump.sh
 
-Bump.sh connects people with APIs. From your API specifications, Bump.sh generates the documentation you need and offers many options to make it your custom and carefully crafted API experience.
+Bump.sh connects people with REST and event-driven APIs, by centralizing a reference point: their documentation, highlighted with the latest changes and updates. Customize their look and feel, manage who can access (read or maintain) which doc.
```
Comment (jayk-u) [inline docs/help/getting-started/index.md:3]:
```suggestion
Bump.sh connects people with REST and event-driven APIs, by centralizing a reference point: their documentation, highlighted with the latest changes and updates. 

From OpenAPI and AsyncAPI specifications, Bump.sh generates the documentation you need and offers many options to make it your custom and carefully crafted API experience.

Those specifications, supported by a wide variety of frameworks in various programming languages that generate contract files automatically, can be easily deployed on Bump.sh in a few clicks.
```

File: docs/help/getting-started/index.md (hunk 4 above)
Comment (jayk-u) [inline docs/help/getting-started/index.md:5]:
```suggestion
```

**Pull request merged**